greenery/fsm_test.py

```diff
@@ -463,6 +463,38 @@ def test_invalid_fsms():
     except Exception:
         pass
 
+    # invalid transition from unknown state
```

Comment:
Go ahead and use the `pytest` assertion here, thanks.

Reply:
Will push that up in a sec

Reply:
That's been rebased and pushed